x/storage/keeper/cross_app.go

```diff
@@ -22,4 +23,10 @@ func RegisterCrossApps(keeper Keeper) {
 	if err != nil {
 		panic(err)
 	}
+
+	permissionApp := NewPermissionApp(permissionKeeper)
```

Comment:
maybe this `permissionApp` can be in `permission` module?

Reply:
it also depends storageKeeper